### PR TITLE
New package: applet-window-buttons-0.11.1

### PR DESCRIPTION
**Pull request by luan-adao**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-GLIBC)
- I built this PR locally for these architectures:
  - armv7l

### CODE REVIEW

File: srcpkgs/applet-window-buttons/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'applet-window-buttons'
+pkgname=applet-window-buttons
+version=0.11.1
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -Wnodev .."
```
Comment (classabbyamp) [inline srcpkgs/applet-window-buttons/template:6]: these are already part of the build style (except for `-Wnodev` which just suppresses some warnings, so we should still leave it out)
```suggestion
```

File: srcpkgs/applet-window-buttons/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+hostmakedepends="extra-cmake-modules gettext qt5-qmake qt5-host-tools kpackage kcoreaddons"
+makedepends="kdecoration-devel plasma-framework-devel"
+short_desc="Plasma 5 applet in order to show window buttons in your panels"
+maintainer="luan-adao <luanadao98@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/psifidotos/applet-window-buttons"
```
Comment (classabbyamp) [inline srcpkgs/applet-window-buttons/template:12]:
```suggestion
homepage="https://github.com/psifidotos/applet-window-buttons"
changelog="https://raw.githubusercontent.com/psifidotos/applet-window-buttons/master/CHANGELOG.md"
```

**Comment by luan-adao** [comment]
@classabbyamp Sorry i made a mistake here, can you reopen the pull request?

**Comment by classabbyamp** [comment]
no, the branch is gone

**Pull request closed without merging**